src/main/scala/com/amazon/deequ/analyzers/QuantileNonSample.scala

```diff
@@ -42,6 +42,22 @@ class QuantileNonSample[T](
 
   expand()
 
+  /** Given sketchSize, shrinkingFactor, and data, reconstruct the KLL Object */
+  def reconstruct(k: Int, c: Double, data: Array[Array[T]]): Unit = {
+    sketchSize = k
+    shrinkingFactor = c
+    compactors = ArrayBuffer[NonSampleCompactor[T]]()
```

Comment:
can we already give this a hint for the right capacity? it seems it needs to hold max(data.indices) values